sway/commands/swap.c

```diff
@@ -84,7 +84,7 @@ static void swap_focus(struct sway_container *con1,
 	}
 }
 
-static void container_swap(struct sway_container *con1,
+void container_swap(struct sway_container *con1,
```

Comment:
nit: since it is now short enough to fit on a single line, can you please combine this and the following line?

Reply:
Done!